Add `The Followpoint: dressurf, The #1 Key Smasher` news post (#11558)
ppy/osu-wiki: news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher.md

---
layout: post

> title: "The Followpoint: dressurf, Top Keyboard Smasher"

Reviewer: bit different from the last one

![image](https://github.com/ppy/osu-wiki/assets/36758269/0980f65a-2171-4297-a510-47a063354e21)

```suggestion
title: "The Followpoint: dressurf, The #1 Key Smasher"
```

date: 2024-06-17 17:00:00 +0000
series: The Followpoint
---

> It's the latest in a series of top player interviews, now featuring the osu!mania #1 best keyboard smasher: [dressurf](https://osu.ppy.sh/users/758406)!

Reviewer: ```suggestion
The last in a series of top player interviews features [dressurf](https://osu.ppy.sh/users/758406), osu!mania's #1 keyboard smasher!
```

![](/wiki/shared/news/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher/banner.jpg)

> That's right, following previous interviews with [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy), [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) and [dressurf](https://osu.ppy.sh/users/758406), top players of osu!, osu!catch, and osu!taiko respectively, this edition of The Followpoint continues the pattern by featuring osu!mania's rank #1 player and the [osu!mania 7K World Cup 2024](/wiki/Tournaments/MWC/2024_7K) champion.

Reviewer 2: ```suggestion
That's right, following previous interviews with [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy), [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) and [Eriha](https://osu.ppy.sh/users/16320311), top players of osu!, osu!catch, and osu!taiko respectively, this edition of The Followpoint continues the pattern by featuring osu!mania's rank #1 player and the [osu!mania 7K World Cup 2024](/wiki/Tournaments/MWC/2024_7K) champion.
```

> That's right, following previous interviews with [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy), [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) and [Eriha](https://osu.ppy.sh/users/16320311), top players of osu!, osu!catch, and osu!taiko respectively, this edition of The Followpoint continues the pattern by featuring osu!mania's rank #1 player and the [osu!mania 7K World Cup 2024](/wiki/Tournaments/MWC/2024_7K) champion.
> 
> Simply read on for a look into [dressurf](https://osu.ppy.sh/users/758406)'s insight:

Reviewer: ```suggestion
The top players of the other game modes have already been interviewed: [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy), [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) and [Eriha](https://osu.ppy.sh/users/16320311), which you may also be interested in reading up on if this one caught your eye.

For now though, the only natural way to proceed is to interview osu!mania's rank #1 player and the [osu!mania 7K World Cup 2024](/wiki/Tournaments/MWC/2024_7K) champion.
```

---

*For the people who don’t know who you are, how would you introduce yourself?*

> Thanks for having me for this event, The Followpoint, my name is dressurf, and I'm constantly striving to improve my skills furthermore.

*How did you get introduced to osu!, and what was your first impression of it?*

> It all began when I played Osu! Tatakae! Ouendan (Elite Beat Agents) on the Nintendo DS Lite. I was immediately intrigued by its unique gameplay style. It didn't take too long to find out that there's a similar rhythm game named "osu!".
>

> > Upon installing it, I remember playing it nonstop for like at least 3 hours on young age. The beatmaps I played then are still can be found on "Most Played Beatmaps" tab on my profile page. It was so fun to play the same style concept game with variety of songs to pick on.

Reviewer: ```suggestion
> Upon installing it, I remember playing it nonstop for like at least 3 hours on young age. The beatmaps I played then are still can be found on "Most Played Beatmaps" tab on my profile page. It was so fun to play a game of the same style and concept with variety of songs to pick between.
```

> *What made you start playing VSRG style games?*

Reviewer: ```suggestion
*What made you start playing VSRG-style games?*
```

> Same as all other players, O2jam introduced me to play VSRG style games.
>
> Come to think of it, I was quite the VSRG enthusiast, trying various rhythm games such as O2jam, Canmusic, Audition, R2Beat, Rhythm Star, Popstage, LR2, Stepmania (Once in middle school days), DJMAX Portable/RAY, Tabsonic, etc. I played so many of them.
>

> > I was really weak player back then, even struggling to play 3 stars (equivalent to the current osu!mania Star Rating) 7K chart.

Reviewer: ```suggestion
> I was a really weak player back then, even struggling to play 3-star (equivalent to the current osu!mania Star Rating) 7K charts.
```

*What other rhythm games do you enjoy, besides games like O2Jam and osu!mania?*

> Sometimes I play DJMAX respect to just ease myself. Trying new key mods, and new gimmicks.
>
> Also, I've been exploring Qwilight, because it's very convenient to play BMS charts and also to find new gimmicks. (Also LR2 is awesome!, by the way!)

*Where does the name "dressurf" come from, and is there any special meaning behind it?*

> I've been anticipating this question from The Followpoint. And yes, there actually is a special meaning behind it.
>

> > When I was using playername as anti-/anti_, I saw myself becoming better steadily at osu!mania. However, I soon realized that my name (This name is) was very common, making it challenging to stand out. So, I decided to change my name to dressurf, which is a combination of my most loved person and my favorite thing when I was a kid.

Reviewer: ```suggestion
> When I was using playername as anti-/anti_, I saw myself becoming better steadily at osu!mania. However, I soon realized that my name (this name) was very common, making it challenging to stand out. So, I decided to change my name to dressurf, which is a combination of my most loved person and my favorite thing when I was a kid.
```

*What would you say is the greatest gift that osu!mania has given you?*

> Playing osu!mania gave lots of things to me, such as managing anxiety, how to schedule things between different timezones and being positive. Most importantly, playing osu!mania made me realize that nothing is impossible unless you try sincerely.

*What kind of maps do you find yourself enjoying to play the most?*

> I really like playing challenging technical charts, they're really fun to play and seeing the combos stack up as the notes fall relentlessly makes me amazing.

> *What’s your favorite score you’ve ever set, and why?*

Reviewer: ```suggestion
*What's your favourite score you've ever set, and why?*
```

> > Without a doubt, Ice angel +DT is the proudest/favorite score I ever set, because it was a turning point for me.

Reviewer: ```suggestion
> Without a doubt, Ice Angel +DT is the proudest/favorite score I ever set, because it was a turning point for me.
```

>
> After finally getting S-rank on that chart, my hand speed and reading abilities improved to new heights. I remember grinding hours and hours to just get a pass on that chart.

*Did you ever see yourself becoming the rank 1 player in osu!mania, why or why not?*

> Not even close, I never saw myself becoming the rank 1 when I was barely passing 6 star charts.
>
> Just like every other player, playing 7K genre was hard. Actually, everything was hard. Coordinating fingers to each column, reading seven keys at once, playing charts even though i don't feel like improving...
>

> > Even watching professional players looked so hard and confusing, but I still had my goal to achieve #1 in the future kept going.

Reviewer: ```suggestion
> Even watching professional players looked so hard and confusing, but I still had my goal to achieve #1 in the future and kept going.
```

*What do you feel like most people don't truly realize about being a top player, especially in the highest ranks?*

> There's undeniable pressure that I always expect and get, such as setting 1900pp's or trying to maintain current spot... So I always practice off-stream to get in shape, exploring pending tab on webpage, downloading new charts, rating them to play on stream later.

*Who would you say are the best osu!mania player(s) at the moment that aren't you, and why?*

> I think the term "best" is very subjective, if I see "best" in terms of inspiring and drawing players into the osu!mania scene, I must say [jhlee0133](https://osu.ppy.sh/users/140148) for 7K genre, [SillyFangirl](https://osu.ppy.sh/users/2288363) for 4K genre.

*Is there any advice that you would give to people who are just starting out, or are wanting to improve to one day be like you?*

> > Well, it's going to be really hard to correlate your fingers to each column, but must be preceded first to get used to 7K playstyle. Once you become accustomed to the 7K playstyle and develop muscle memory for finger positioning, your progress will accelerate.

Reviewer: ```suggestion
> Well, it's going to be really hard to correlate your fingers to each column, but that must be achieved first to get used to the 7K playstyle. Once you become accustomed to the 7K playstyle and develop muscle memory for finger positioning, your progress will accelerate.
```

*Do you think that peripherals matter when it comes to becoming better at the game, why or why not?*

> Yes, if you use a keyboard that's high, I strongly recommend utilizing a wrist rest or at least a towel to alleviate potential wrist pain.
>

> > A minimum requirement for keyboard I think is the ability to press 7+ keys properly at once. Additionally, 144hz monitor would be great advantage if you're going to step up into more extreme charts.

Reviewer: ```suggestion
> A minimum requirement for keyboard I think is the ability to press 7+ keys properly at once. Additionally, 144hz monitor would be a great advantage if you're going to step up into more extreme charts.
```

*What drives you to keep improving despite being at the top?*

> > Until now, and still, I always set myself as rival and competed against myself, consistently challenging my previous scores to improve. I deliberately make charts' rate a little bit over my playlimit and play them when my hand's condition is very good. Of course, I get 91~92% on it, but I primarily concentrate on getting scores to battle on later.

Reviewer: ```suggestion
> Until now, and still, I always set myself as the rival and competed against myself, consistently challenging my previous scores to improve. I deliberately make charts' rate a little bit over my playlimit and play them when my hands' condition is very good. Of course, I get 91~92% on it, but I primarily concentrate on getting scores to battle on later.
```

*What are some "endgame" scores or milestones you would want to achieve in the future?*

> I want to develop at least 300+ bpm chordjack, 270~280+ bpm brackets or passing Stellium regular rice dan on 1.1x.

*To finish things up, what does the future hold for you? Do you have any goals or anything else that you'd like to mention?*

> > First and foremost, I really want to thank all the supporters who have been by my side and those supports really drives me to continually improve more.

Reviewer: ```suggestion
> First and foremost, I really want to thank all the supporters who have been by my side. The support really drives me to continually improve more.
```

>

> > It'll be really hard to play 7K genre at first, but trust me, they're really fun mod if you get used to playing in those columns! Never hesitate to seek advices from professional players, they'll gladly guide you and help you as you move on. Currently, my message box (DM) is closed, so you can ask me anything on Discord server by mentioning my name.

Reviewer: ```suggestion
> It'll be really hard to play 7K at first, but trust me, it's a really fun mode if you get used to playing in those columns! Never hesitate to seek advice from professional players, because they'll gladly guide you and help you as you move on. Currently, my message box (DM) is closed, so you can ask me anything on Discord server by mentioning my name.
```

>

> > Thanks for having me on Followpoint, I hope these answers cleared up your questions!

Reviewer: ```suggestion
> Thanks for having me on The Followpoint, and I hope these answers cleared up your questions!
```

---

Huge thanks to [dressurf](https://osu.ppy.sh/users/758406) for joining us and answering these questions! Once again, I hope you as a reader enjoyed this interview. <3

> We hope you enjoyed this series of us interviewing the top players in each gamemode, it was a lot of fun for us, and we can't thank you enough for the support! We will have another interivew come out in the next two weeks, but we will keep it a surprise on who it is, so you'll just have to guess. You can also follow [@The_Followpoint on Twitter](https://twitter.com/The_Followpoint) for updates and question submissions for future interviews.

Reviewer: ```suggestion
We hope you enjoyed this series of us interviewing the top players in each game mode, it was a lot of fun for us, and we can't thank you enough for the support! We will have another interview come out in the next two weeks, but we will keep it a surprise on who it is, so you'll just have to guess. You can also follow [@The_Followpoint on Twitter](https://twitter.com/The_Followpoint) for updates and question submissions for future interviews.
```
